src/router.js

```diff
@@ -15,20 +18,19 @@ export default function(app) {
     },
     events: {
       loaded: function(state, actions) {
-        addEventListener("popstate", function() {
+        match()
```

Comment:
humm..  I see what you are saying, have you tried testing it locally to see if you get the same results?